testkit/src/main/java/org/apache/calcite/test/CalciteAssert.java

```diff
@@ -1736,15 +1736,6 @@ public AssertQuery queryContains(Consumer<List> predicate1) {
       });
     }
 
-    // CHECKSTYLE: IGNORE 1
-    /** @deprecated Use {@link #queryContains(Consumer)}. */
-    @SuppressWarnings("Guava")
-    @Deprecated // to be removed before 2.0
-    public final AssertQuery queryContains(
-        com.google.common.base.Function<List, Void> predicate1) {
-      return queryContains((Consumer<List>) predicate1::apply);
-    }
```

Comment:
With guava 32 it starts failing on errorprone checks with `CheckReturnValue`
Since it is already deprecated with  a comment what to use instead probably it makes sense to remove it